### DISCUSSION TITLE
Mass material flow instead of number of trips?

### DISCUSSION THREAD

**Discussion by lamarkle** [FlowmapBlue/FlowmapBlue#91] [Q&A] [answered]
Is there a potential route to change the label of flows from number of trips to something like material tonnage? I am interested in looking at material flows (recycling) rather than number of trips. Thank you!!

**Comment by ilyabo** [comment, marked as answer]
Yes, you can customize the labels in the properties sheet of the spreadsheed:

<img width=500 src=https://github.com/FlowmapBlue/FlowmapBlue/assets/351828/992b736e-60f3-407f-87f5-a0858d43145a>

**Reply by lamarkle** [reply]
Thank you so much, sorry I missed this! I was also wondering if you have a suggested or preferred way to embed maps on websites? Thank you!

**Reply by ilyabo** [reply]
Yes, you can generate the embed code if you click on the "Share" button:

<img src=https://github.com/FlowmapBlue/FlowmapBlue/assets/351828/81deda1a-3e80-46b3-ad8c-38d19e2cc605 width=500>

**Reply by lamarkle** [reply]
Thank you! Is this possible on R Studio as well?
